src/Symfony/Component/Notifier/Bridge/Ntfy/README.md

```diff
@@ -5,16 +5,24 @@ Provides [Ntfy](https://docs.ntfy.sh/) integration for Symfony Notifier.
 
 DSN example
 -----------
-
 ```
-NTFY_DSN=ntfy://[USER:PASSWORD]@default[:PORT]/TOPIC?[secureHttp=[on]]
+NTFY_DSN=ntfy://URL[:PORT]/TOPIC?[secureHttp=[on]]
+```
+or with username and password authorization:
+```
+NTFY_DSN=ntfy://USER:PASSWORD@URL[:PORT]/TOPIC?[secureHttp=[on]]
+```
+or with token authorization:
+```
+NTFY_DSN=ntfy://:TOKEN@URL[:PORT]/TOPIC?[secureHttp=[on]]
 ```
 where:
 - `URL` is the ntfy server which you are using
-    - if `default` is provided, this will default to the public ntfy server hosted on [ntfy.sh](https://ntfy.sh/).
```

Comment:
Looking at README in other bridges, the example DSN uses `@default` directly. I think it'd be better to follow their lead as using something non-default is going to be the exception.

Reply:
Ok, I understand. But text is talking about `URL` which is not in example DSN. With self-hosted Ntfy is `default` unclear